src/angular/styles/_variables.scss

```diff
@@ -84,10 +84,6 @@ $sbbColorPeach: #fcbb00;
 $sbbColorLemon: #ffde15;
 $sbbColorGreen: #00973b;
 
-// Non-Standard Colors
-$sbbColorGrey: #666666;
-$sbbColorGreyMedium: #979797;
-
```

Comment:
This is a breaking change for me which should we better not commit right now. Maybe also leave it here forever.